src/SmartComponents/Systems/SystemsMainContent.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { Main } from '@redhat-cloud-services/frontend-components/Main';
+import { useSelector, useDispatch } from 'react-redux';
+import messages from '../../Messages';
+import ErrorHandler from '../../PresentationalComponents/Snippets/ErrorHandler';
+import { DEFAULT_PATCH_TITLE } from '../../Utilities/constants';
+import { changeSystemsParams, clearInventoryReducer } from '../../store/Actions/Actions';
+import { usePatchSetState } from '../../Utilities/hooks';
+import { intl } from '../../Utilities/IntlProvider';
+import { decodeQueryparams } from '../../Utilities/Helpers';
+import { useActivateRemediationModal } from './SystemsListAssets';
+import SystemsStatusReport from '../../PresentationalComponents/StatusReports/SystemsStatusReport';
+import PatchSetWrapper from '../../PresentationalComponents/PatchSetWrapper/PatchSetWrapper';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import RemediationWizard from '../Remediation/RemediationWizard';
+import SystemsTable from './SystemsTable';
+import { useSearchParams } from 'react-router-dom';
+
+const SystemsMainContent = () => {
+    const chrome = useChrome();
+    useEffect(()=>{
+        chrome.updateDocumentTitle(`${intl.formatMessage(messages.titlesSystems)}${DEFAULT_PATCH_TITLE}`);
```

Comment:
The `SystemsPage` below does update the title as well. Are they both required, or could one of them be removed?

Reply:
Yup, this is not intended to be here. Will remove it from SystemsMainComponent.